### PR TITLE
Improve tutorial documentation; fixes #71

### LINKED ISSUE #71: Add clarity to Tutorial step 3
This issue was originally reported by @hori031617 via facebook/react/issues/10323

### Text of original issue:

I tried tutorial.

> 3. Delete all files in the src/ folder of the new project.

I think that good example code here.
For example `rm -f src/*`
Because, I had a misunderstanding remove src directory.

Thank you.

### PR DESCRIPTION
**Pull request by tricinel**

### CODE REVIEW

**Comment by bvaughn** [comment]
Hey @tricinel. Thanks for the PR!

Unfortunately PR #154 was filed before this one, and so I merged it this morning. (I try to merge the earliest PR.)

It's a subset of the changes you propose, and I'm still happy to review yours, but you'll need to rebase to fix conflicts. 😄

**Comment by tricinel** [comment]
Yep, understandable :)

I'll review in a sec.

**Comment by tricinel** [comment]
@bvaughn, that should be it I hope :)

File: content/tutorial/tutorial.md [hunk 1]
```diff
@@ -54,9 +54,33 @@ If you want to do it, here are the steps to follow:
...
+```bash
+touch src/index.css
+curl -L https://codepen.io/gaearon/pen/oWWQNa.css > src/index.css
+```
+
+5. Add a file named `index.js` in the `src/` folder with [this JS code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010), or type in the following commands:
+
+```bash
+touch src/index.js
+curl -L https://codepen.io/gaearon/pen/oWWQNa.js > src/index.js
```
Comment (bvaughn) [inline content/tutorial/tutorial.md]: I just noticed that this JS is compiled. That's probably not ideal.

I'm thinking maybe we should just remove these 2 touch+curl snippets, at least unless there's a way to get the pre-compilation JS. Thoughts?

Comment (tricinel) [reply content/tutorial/tutorial.md]: Hm, yes, you're right. Should have paid more attention to that. Do you feel we should remove the touches as well, or just the curls?

Comment (bvaughn) [reply content/tutorial/tutorial.md]: Eh...probably both? Might be confusing to have the `touch` without the `curl`.

Comment (tricinel) [reply content/tutorial/tutorial.md]: Cool. I'll remove in a sec.

**Comment by bvaughn** [comment]
Thanks~

**Pull request merged**